CRM/Import/DataSource/CSV.php

```diff
@@ -253,10 +253,10 @@ private static function _CsvToTable(
    * See also dev/core#2127 - avoid breaking strings ending in à or any other
    * unicode character sharing the same 0xA0 byte as a non-breaking space.
    *
-   * @param string $string
+   * @param string $string Allow null values from empty rows
    * @return string The trimmed string
    */
-  public static function trimNonBreakingSpaces(string $string): string {
+  public static function trimNonBreakingSpaces(?string $string): string {
```

Comment:
Ah I see.

So it then treats it as ASCII, and then converts it to the empty string. Maybe we should also be explicit right at the start of the function, to avoid future weirdness.
`if ($string === NULL) { return ''; }`